### PR TITLE
PREOPS-3431: address various minor items such as typos in the docs

### PR DESCRIPTION
**Pull request by ameisner**
This is one piece of the Jira ticket PREOPS-3431. I still need to identify/assign a reviewer, but in any case I expect that review will be very quick. Thanks very much...

### CODE REVIEW

File: getting-started/data-setup.rst
Comment (gpdf) [inline getting-started/data-setup.rst:125]: Maybe "exposure" - in quotes - and use quotes consistently throughout the next couple of paragraphs to highlight the terms being defined?

File: getting-started/display.rst
Comment (gpdf) [inline getting-started/display.rst:175]: "different mask bit that corresponds to detections" could be misread as "each detection has its own mask bit" or something similarly off-track.

Maybe:

Each color corresponds to one of a common set of boolean flags defined for each pixel; each flag has a distinct meaning, covering a range of concepts from detector artifacts to the contribution of a pixel to a detection.  In this project we call the array of these flags the "mask" or "mask plane" associated with an image.  (NB: This is a distinct  concept from a "mask" for an array in ``numpy``.)  You'll learn later how to interpret these flags (and their colors); for now you'll likely want to adjust the image display itself.

**Review by gpdf — COMMENTED** [review]
A couple of suggestions for further improvement...

File: getting-started/data-setup.rst
Comment (ameisner) [reply getting-started/data-setup.rst]: Good idea. Should be addressed in [425035a](https://github.com/lsst/pipelines_lsst_io/pull/292/commits/425035a870e3602be874460b9bf0c511ba92f8a5), but let me know if not. Thanks...

File: getting-started/display.rst
Comment (ameisner) [reply getting-started/display.rst]: Thanks for the suggested rewording! I've implemented this in [c220b3a](https://github.com/lsst/pipelines_lsst_io/pull/292/commits/c220b3addec6cc7bbe57d3b2924a6109d0c1bed2)

**Comment by ameisner** [comment]
@gpdf -- thanks for the detailed comments! I believe that I've implemented them via added commits on this same branch. Could you please take a look and approve this PR if appropriate? Thanks!

File: getting-started/display.rst
Comment (gpdf) [inline getting-started/display.rst:175]: Tiny quibble: can you use "line break after every sentence" style here, as has largely been done elsewhere in the document?  It facilitates future editing and change-tracking.

File: getting-started/coaddition.rst
Comment (gpdf) [inline getting-started/coaddition.rst:120]: While we're in here... The use of "warp" as a noun is clearly jargon.  At the start of this file "warp" is sort of defined, as a verb: "... warp (retroject) ..." (currently line 17).  Could we use quotation marks there, at the point of first definition (for consistency with the other changes we've made) and then here (on line 120) just say "... find the warped images from ..."?

Finally, I haven't read the whole tutorial thoroughly, but I'm wondering whether there's any place where we're giving people the mapping between the DatasetType strings and the English/jargon terms that we're using for the different intermediate and final products discussed in this documentation.

Comment (gpdf) [reply getting-started/coaddition.rst:120]: There's a brief mention of `calexp` and `raw` in part 3 (Display) but I think the tutorial would be a lot clearer with more said about this.  It would make Figure 1 in part 2 (Single Frame) easier to understand, I think, too.

**Review by gpdf — COMMENTED** [review]
I've made a few more comments now that I am recognizing that I'm being counted on for the "real" review.  ;)

The main one is that I think the getting-started tutorial would be improved by a more explicit introduction of the concept of DatasetTypes and their mapping to the concepts mentioned in the various sections of intermediate (e.g., warps) and final (e.g., calexp) data products.

So, sorry, not quite "Approve" yet, but if the CST wants to reclaim or give the review to a pipelines person, I won't object.

**Comment by ameisner** [comment]
> I've made a few more comments now that I am recognizing that I'm being counted on for the "real" review. ;)

I'll continue attempting to implement all of your suggestions for now :). I am slightly unclear on how you became a/the reviewer (did you mark yourself as a reviewer? I don't _think_ that I marked you as the reviewer).

File: getting-started/display.rst
Comment (ameisner) [reply getting-started/display.rst]: Implemented in [c49b6fd](https://github.com/lsst/pipelines_lsst_io/pull/292/commits/c49b6fd98585876cf4171f8a84d828bd87219af4).

File: getting-started/coaddition.rst
Comment (ameisner) [reply getting-started/coaddition.rst]: The detailed edits suggested in this conversation have now been implemented in [4956737](https://github.com/lsst/pipelines_lsst_io/pull/292/commits/4956737533bc7df8497e373e996353c14da3b230).

Comment (ameisner) [reply getting-started/coaddition.rst]: I have now added an explanation/introduction of the notion of `DatasetType` in section 1 of the getting started tutorial ([1a99f3b](https://github.com/lsst/pipelines_lsst_io/pull/292/commits/1a99f3b62c68a7d7912f110473c244c9043e77a7)). I note that, although the concept of a "warp" is mentioned variously throughout this getting started tutorial, I did not find any actual mention of a specific ``DatasetType`` associated with the warps produced (I believe there are multiple different ``DatasetTypes`` that could correspond to warps).

**Comment by ameisner** [comment]
@gpdf -- I believe I've now addressed/implemented all of your additional comments/suggestions. Could you please check and let me know whether this PR is approved or requires additional changes? Thanks again...

**Comment by gpdf** [comment]
> > I've made a few more comments now that I am recognizing that I'm being counted on for the "real" review. ;)
> 
> I'll continue attempting to implement all of your suggestions for now :). I am slightly unclear on how you became a/the reviewer (did you mark yourself as a reviewer? I don't _think_ that I marked you as the reviewer).

No, I just saw the PR and was curious enough about it to post some comments.  I think really @MelissaGraham should probably decide whether this is sufficient review or whether someone from DRP should also look at this.

**Review by gpdf — APPROVED** [review]
I'm happy with these changes now.  As noted above, the CST should decide whether my review actually filled the gap of what it was looking for.

**Review by MelissaGraham — APPROVED** [review]
Mostly just typo fixes, approved.

**Pull request merged**
